### PR TITLE
ENH: Adds baseline random kek searching function

### PR DESCRIPTION
**Pull request by SantiagoTorres**
1) Gets a random word from wikipedia

2) Searches for a list of such word

3) Reports back with results if there's not a list of it :[

An example usage is hardcoded in the main function of src/check_list.py

### CODE REVIEW

File: src/check_list.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    <Description>
+        Obtains a random word from /usr/share/words and checks
+        if there is a list of that. 
+
+    <Usage>
+        from check_list import check_random_list, LOLException
+        try:
+            check_random_list()
+        Except LOLException as e:
```
Comment (ElDeveloper) [inline src/check_list.py]: Except -> except

File: src/check_list.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```
Comment (ElDeveloper) [inline src/check_list.py]: We shouldn't really need this. Let's not make this both module and a script, you agree? or alternatively if you think it is too early to think about that, I'm OK.

File: src/check_list.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            print("No new list-less object for LOL")
+""" 
+import wikipedia
+import linereader
+import random
+
+DICT = "../dict.txt"
+WC_DICT = 48915
+
+class LOLException(Exception):
```
Comment (ElDeveloper) [inline src/check_list.py:26]: Took me a little bit to understand the significance of this, for it is an exception, nay an error if you find an object without a list! Would you mind describing that in the docstring of this exception? i.e. raised when .... etc

File: src/check_list.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    randfile = linereader.copen(DICT)
+    random_word = randfile.getline(random.randint(1, WC_DICT))
+
+    try:
+        query = "List of {}".format(random_word)
+        page = wikipedia.page(query)
+        result_word = page.title.lstrip("List of")
+        if result_word != random_word:
+            raise LOLException(random_word)
+    except wikipedia.exceptions.PageError:
```
Comment (ElDeveloper) [inline src/check_list.py:63]: Not too familiar with Wikipedia's API yet, but is it possible to remove some of these lines from the try/except block, for example `query = ....` doesn't really need to be there, right? are there other instances like that? i.e for which we aren't expecting a `PageError`?

File: src/check_list.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        raise LOLException(random_word)
+    
+    return random_word
```
Comment (ElDeveloper) [inline src/check_list.py:70]: Would be good to describe this in the docstring of the function.

**Comment by ElDeveloper** [comment]
Thanks @SantiagoTorres, I think this looks good overall, a few minor comments, mostly trying to roughly figure out where we are headed.

---

> 3) Reports back with results if there's not a list of it :[

This is probably one of the saddest sentences I've seen in GitHub.

File: src/check_list.py (hunk 2 above)
Comment (SantiagoTorres) [inline src/check_list.py]: True

File: src/check_list.py (hunk 1 above)
Comment (SantiagoTorres) [inline src/check_list.py]: True

File: src/check_list.py (hunk 3 above)
Comment (SantiagoTorres) [inline src/check_list.py:26]: Let me give it a shot

**Comment by SantiagoTorres** [comment]
@ElDeveloper I tried to address these, let me know what you think

**Comment by ElDeveloper** [comment]
Danke herr! :+1:

**Pull request merged**